lib/active_record/sql_analyzer/configuration.rb

```diff
@@ -150,15 +150,14 @@ def setup_defaults
         @options[:sql_redactors] = [
           Redactor.new(/\n/, " "),
           Redactor.new(/\s+/, " "),
-          Redactor.new(/IN \([^)]+\)/i, "IN ([REDACTED])"),
-          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?-?\d+(\.\d+)?/i, " = [REDACTED]"),
+          Redactor.new(/IN \([^)]+\)/i, "IN ('[REDACTED]')"),
+          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?-?\d+(\.\d+)?/i, " = '[REDACTED]'"),
           Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?x?#{quotedValuePattern}/i, " = '[REDACTED]'"),
-          Redactor.new(/VALUES \(.+\)$/i, "VALUES ([REDACTED])"),
+          Redactor.new(/VALUES \(.+\)$/i, "VALUES ('[REDACTED]')"),
           Redactor.new(/BETWEEN #{quotedValuePattern} AND #{quotedValuePattern}/i, "BETWEEN '[REDACTED]' AND '[REDACTED]'"),
-          Redactor.new(/LIKE #{quotedValuePattern}/i, "LIKE '[REDACTED]'"), 
+          Redactor.new(/LIKE #{quotedValuePattern}/i, "LIKE '[REDACTED]'"),
           Redactor.new(/ LIMIT \d+/i, ""),
           Redactor.new(/ OFFSET \d+/i, ""),
-          Redactor.new(/INSERT INTO (`?\w+`?) \([^)]+\)/i, 'INSERT INTO \1 ([COLUMNS])'),
```

Comment:
If the output is a mess without the column list redaction, I'll look at a better way to deal with it.